builder/builder.go

```diff
@@ -332,16 +332,17 @@ func GetBuilders(dockerCli command.Cli, txn *store.Txn) ([]*Builder, error) {
 }
 
 type CreateOpts struct {
-	Name       string
-	Driver     string
-	NodeName   string
-	Platforms  []string
-	Flags      string
-	ConfigFile string
-	DriverOpts []string
-	Use        bool
-	Endpoint   string
-	Append     bool
+	Name                string
+	Driver              string
+	NodeName            string
+	Platforms           []string
+	BuildkitdFlags      string
+	BuildkitdConfigFile string
+	BuildkitdNetmode    string
```

Comment:
For me a driver-opt is driver specific (container config for docker-container with docker api, pod specific for k8s api) while `buildkitd-(flags|config)` is common. I think this would be confusing to put it as driver-opt.

Reply:
I don't think this is an important enough case to have a special option. Basically, it is only a test option for v0.13. After v0.14, if bridge becomes default we don't expect anyone to ever use this anymore.

Reply:
> After v0.14, if bridge becomes default we don't expect anyone to ever use this anymore.

If we have more network mode in the future that could be valuable I think? Like a `wg` mode (wireguard) one?

If you don't see this as valuable to keep, I was thinking we could use an env var instead? Something like `BUILDKIT_NETMODE` or `EXPERIMENTAL_BUILDKIT_NETMODE`? But current vars https://docs.docker.com/build/building/variables/#build-tool-configuration-variables are not drive for `create` cmd, only build or bake so might not be ideal :disappointed:

So yeah maybe a driver-opt with a specific description about it saying this is temporary could be ok.

Reply:
Maybe we should skip it then and just document setting `--buildkitd-flags`. We can still keep the test.

Reply:
> Maybe we should skip it then and just document setting `--buildkitd-flags`. We can still keep the test.

Yeah `--buildkitd-netmode` is more likely a shorthand for `--buildkitd-flags="--oci-worker-net=<mode>"`. I'm fine just documenting `--buildkitd-flags` to set custom network mode.